File: IntegracaoPython/antecipacao.py
import pagarme
import json
import time

def Antecipacao():

    pagarme.authentication_key('ak_test_jbIXhrHXHOaUUNKtUVrkT9HGL60SSg')

    params = {
        'payment_date': time.time()*1000 + 86400000,
        'timeframe': 'start',
        'requested_amount': '5000'
    }
    recipientId = 're_ck0zjmw1s007ier6eysyd3agj'
    bulk_anticipation = pagarme.bulk_anticipation.create(recipientId, params)

    ante_in_string = json.dumps(bulk_anticipation)
    parsed_json = (json.loads(ante_in_string))

    return parsed_json["id"], parsed_json["amount"], recipientId

File: IntegracaoPHP/antecipacao.php
<?php
require("vendor/autoload.php");
$pagarme = new PagarMe\Client('ak_test_jbIXhrHXHOaUUNKtUVrkT9HGL60SSg');

$recipientId = "re_ck0zjmw1s007ier6eysyd3agj";
$anticipation = $pagarme->bulkAnticipations()->create([
    'recipient_id' => $recipientId,
    'build' => 'true',
    'payment_date' => time()*1000+604800000,
    'requested_amount' => '1000',
    'timeframe' => 'start'
]);
echo("A antecipação: ".$anticipation->id.", de ".$anticipation->amount." centavos foi criada para o recebedor ".$recipientId."!");